src/frontend/screens/Game/GamePage/index.scss

```diff
@@ -218,7 +218,7 @@
       margin-bottom: 0;
     }
 
-    svg {
+    svg:not(:is(button > svg)) {
```

Comment:
it looks like we can remove this whole selector+style instead of changing the selector

I don't see anywhere in that page where text cursor makes sense, like... the icons next to Installed Info / Game Scores / Proton Compatibility / Time Played / Play Now / Report a problem...  are also all clickable

Reply:
Yeah I thought so too, but didn't want to change some sort of functionality here just _in case_ it was needed. 🙂 

I'll remove it altogether so - thanks!